### ISSUE TITLE
Using Redis as a persistent store

### ISSUE THREAD

**Issue by knzudgt** [dokku/dokku-redis#70]
I would need Redis for Sidekiq.
Sidekiq's documentation suggests that "it's important that Sidekiq be run against a Redis instance that is not configured as a cache but as a persistent store".
Is dokku-redis configured to run as a cache or as a persistent store?

**Comment by josegonzalez** [comment]
Whats the difference?

**Comment by knzudgt** [comment]
I found the recommendation to use Redis as persistent store and not as cache at the official Sidekiq GitHub repository, in the [wiki section](https://github.com/mperham/sidekiq/wiki/Using-Redis#memory).
I am trying to understand the difference myself, but the most important point is that, as far as I understood, using Redis as a cache, as usually people do, would not provide safety against data loss.
If Redis is used as persistent store, Redis produces point-in-time snapshots of the dataset that works like backups which garantee against data corruption, and (if the Append Only File is used in conjunction with snapshotting) "logs every write operation received by the server, that will be played again at server startup, reconstructing the original dataset." (See [Redis documentation](https://redis.io/topics/persistence)).
I found a very interesting [article](http://oldblog.antirez.com/post/redis-persistence-demystified.html) that I would suggest to anyone who is interested in the topic, and that I am personally struggling to understand being myself not an advanced user.
